exercises/bowling/BowlingTest.cs

```diff
@@ -60,7 +60,30 @@ public void Consecutive_spares_each_get_a_one_roll_bonus()
     public void A_spare_in_the_last_frame_gets_a_one_roll_bonus_that_is_counted_once()
     {
         var sut = new BowlingGame();
-        var previousRolls = new [] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7, 3, 7 };
+        var previousRolls = new[]
```

Comment:
Why are these all on separate lines but the previous tests aren't? IMO it's probably better to have it on one line

Reply:
I guess there are a few other occurrences in the rest of this test file.

Reply:
Well, this is me trying to be clever. Many of the input arrays have a very limited number of elements (e.g. 1 or 2), in which case rendering them over multiple lines would be overkill IMHO. The code that does this check can be found [here](https://github.com/exercism/csharp/pull/601/files#diff-186289da388a45f86733f30d74d153dcR75). It essentially checks if it is not an array of arrays and if the rendered single line output would be less than or equal to 72 characters. If so, it is output as a single line. 

This is of course completely up for debate. I'd be happy to hear your thoughts on this.

Reply:
What I could do, is detect how many lines would end up being created. If there are more than a given number, say 8, then I could put multiple elements on a single line. Would that be an option?

Reply:
@robkeim I've given it some more thought, and I think I was attempting to be a bit too clever. I've removed the automatic multi-line rendering of single-dimensional arrays. Exercises can choose to opt-in themselves, should they want to do multi-line array initialization.